libs/agent-runner.js

```diff
@@ -353,6 +440,22 @@ agent.prototype.runPushOperation = function runPushOperation(operation) {
         return;
       }
 
+      if (encryptionKey && commits.length) {
+        log.info('[ENCRYPTION] Encrypting data before it\'s committed...');
```

Comment:
should this be 'uploaded' instead of 'committed'?

Reply:
No. Data is committed (see commit endpoint).